bundle/python_bundle_workflow/scripts/inference.py

```diff
@@ -140,32 +134,27 @@ def _evaluator(self):
             amp=True,
         )
 
-    @property
     def _handlers(self):
         return [
             # use the logger "eval_log" defined at the beginning of this program
             StatsHandler(name="eval_log", output_transform=lambda x: None),
-            CheckpointLoader(load_path=self.bundle_root + "/models/model.pt", load_dict={"net": self._network_def}),
+            CheckpointLoader(load_path=self.bundle_root + "/models/model.pt", load_dict={"net": self.network_def}),
         ]
 
-    @property
     def _dataset(self):
         return Dataset(
-            data=self._dataset_data,
+            data=self.dataset_data,
```

Comment:
Shouldn't this be `self._dataset_data` ?

Reply:
Hi @vfdev-5 ,

Here we call the `dataset_data` property, and the `_get_property()` function will call the `def _dataset_data()`.

Thanks.